docs/standard/datetime/snippets/timeprovider-overview/csharp/CalendarHelper.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace myproject;
```

Comment:
```suggestion
namespace ExampleProject;
```

Or something that's following namespace best practices, correct casing.